kogitoq/gadaq/pom.xml

```diff
@@ -58,6 +76,11 @@
       <artifactId>rest-assured</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+			<groupId>com.squareup.okhttp3</groupId>
+			<artifactId>okhttp</artifactId>
```

Comment:
What is this package used for? Can't we just use native java packages.

Reply:
This is for minIO client to make network calls.
Quarkus also uses this dependency but has a lower version in its bom so, override with this one.